test/release/examples/benchmarks/shootout/mandelbrot.prediff

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Compare mandlebrot program output.
+
+# Due to numerical instability with different optimization
+# levels, this script tolerates a small number of bit differences
+# in the output.
+
+import os
+import os.path
+import socket
+import sys
+
+testname=sys.argv[1]
+testout=sys.argv[2]
+
+diffs_to_allow=4
+
+clbg_expect=testname+"clbg.pbm"
+expect=testname+".pbm"
+
+hostname=socket.gethostname();
+
+error = ""
+
+if os.getenv('CHPL_TEST_PERF') == 'on':
+  # do nothing for performance tests
+  # these look for data after the .pbm and generally generate
+  # more image data than the expected .pbm
+  exit()
+
+if not os.path.isfile(expect):
+  error = "Error - could not find expected output " + expect
+else:
+
+  if hostname.startswith("chapel-shootout"):
```

Comment:
Let's just do `if 'shootout' in hostname` since it seems likely that even if the machine gets renamed again it'll still have shootout in the hostname.